native/response/response.go

```diff
@@ -21,4 +21,6 @@ type Response struct {
 	// executed at impression time, where it can be supported.
 	JSTracker string          `json:"jstracker,omitempty"`
 	Ext       json.RawMessage `json:"ext,omitempty"`
+	// Array of tracking objects to run with the ad.
```

Comment:
Unnecessary comment

Reply:
Each field has comment as in RTB documentation, think it's need keep